src/helpers.ts

```diff
@@ -0,0 +1,62 @@
+export interface Player {
```

Comment:
You can use `OASModel` from feTS.

Reply:
like that also make a errors:

```
type Player = OASModel<
  NormalizeOAS<(typeof openapi)["definitions"]["Player"]["properties"]>,
  "Player"
>;
```

@ardatan

Reply:
Could you try with `fets@0.4.8`?

Reply:
@ardatan 
Its work!!! thank you Arda❤️